web_src/js/features/migration.js

```diff
@@ -27,6 +29,11 @@ function checkAuth() {
 }
 
 function checkItems(tokenAuth) {
+  if ($mirror.is(':checked')) {
+    $items.addClass('disabled');
+    return;
+  }
```

Comment:
Will this need an else in order to remove the `disabled` class if the mirror checkbox is de-selected?

Reply:
no because it just does the normal check for a token below to decide if it should remove that. If you uncheck mirror and there is no token it should still have items disabled.

Reply:
Ah, I see. Had to expand the blob, now it makes sense. 😄

Reply:
: )